overcloud.yml

```diff
@@ -27,6 +27,19 @@
           src: "firstboot-nvme.yaml.j2"
           dest: "/home/stack/firstboot-nvme.yaml"
         when: nvme is defined
+      - block:
+          - name: adjust max number of concurrent builds to default (override infrared)
+            ini_file:
+                path: /var/lib/config-data/puppet-generated/nova/etc/nova/nova.conf
+                section: DEFAULT
+                option: max_concurrent_builds
+                value: 10
```

Comment:
Instead of hardcoding, we can parameterize this value.

Reply:
Probably not, I don't see us changing this value very often. If we need to let's do it at that point.